tutorials/conf.py

```diff
@@ -96,7 +96,12 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = 'astropy_favicon.ico'
+html_favicon = "_static/astropy_favicon.ico"
```

Comment:
This shouldn't be changed: if you look at the comment above this block, it says "the name of an image file (within the static path)", so in this case you don't need to preface it with `_static`

Reply:
I tried it without the `_static/` path and an error was thrown. And I double-checked on the Internet and someone mentioned that it should be relative to the confdir location, and not relative to the `_static/` path... Could you double check for me, maybe it has to do with the `html_path_static` path defined after this line? I built the docs locally and if I do what you are suggested this file cannot be found.

Reply:
Strange - this does remove the WARNING but I don't understand why it seems to work even without it. In any case, keep it!

Reply:
Sure, no problem!